distribution/server-dist/src/main/docs/examples/map-storage-concurrenthashmap.cli

```diff
@@ -0,0 +1,35 @@
+##
+## CLI script to set Keycloak to use map storage rather than the standard JPA.
+## The backend database is at this moment a ConcurrentHashMap-based storage
+## which is suitable for dev and testing in standalone node. It does not
+## support clustered deployments.
+##
+## Apply this file using the following command from the Keycloak root directory:
+##
+##   bin/jboss-cli.sh --file=docs/examples/map-storage-concurrenthashmap.cli
+##
+## This will modify standalone/configuration/standalone.xml
+##
+
+embed-server
+
+/subsystem=keycloak-server/spi=authorizationPersister:add(default-provider=map)
```

Comment:
Should we add a possibility to configure default-provider using env variable similarly as we have for undertow?

Something like:
```suggestion
/subsystem=keycloak-server/spi=authorizationPersister:add(default-provider=${keycloak.authorization.provider:map})
```

This way it is easier to use map storage only for `userSessions` for example.

Reply:
Given the file name (`map` storage), I would prefer keeping it as is.